### PR TITLE
Added Circle

### PR DESCRIPTION
**Pull request by yktemp1**
It displays a circle to screen of radius 20pixels

### CODE REVIEW

File: scripts/main.js [hunk 1]
```diff
@@ -5,4 +5,5 @@ function setup(){
 
 function draw(){
     background("#333");
-}
\ No newline at end of file
+    circle(width/2, height/2, 20,20);
```
Comment (yktemp2) [inline scripts/main.js]: circle function accepts only two parameters.
Here you should delete last parameter

**Review by yktemp2 — CHANGES_REQUESTED** [review]
@yktemp1 Review these changes before merging

File: scripts/main.js (hunk 1 above)
Comment (yktemp1) [reply scripts/main.js]: Thanks for pointing that

Comment (yktemp1) [reply scripts/main.js]: Now I have fixed that

**Comment by yktemp2** [comment]
Why Did you unmerged

**Comment by yktemp1** [comment]
@yktemp2 Can You add a rectangle instead

**Comment by yktemp2** [comment]
> @yktemp2 Can You add a rectangle instead

Sure

**Pull request merged**